docs/HowTo/Builder-Network.md

```diff
@@ -0,0 +1,128 @@
+Enable registration for all validators using [`--validators-builder-registration-default-enabled`](../Reference/CLI/CLI-Syntax.md#validators-builder-registration-default-enabled)
+
+To enable registration for specific validators only, use the
+[--validators-proposal-config](../Reference/CLI/CLI-Syntax.md#validators-proposer-config) option
+and specify the enabled validators in the `proposer_config` of the proposer configuration JSON file.
+
+Note the `default_config` applies to all validators who don't have their own proposer configuration.
+
+!!! example "`proposerConfig.json`"
+
+    ```json
+    {
+      "proposer_config": {
+        "0xa057816155ad77931185101128655c0191bd0214c201ca48ed887f6c4c6adf334070efcd75140eada5ac83a92506dd7a": {
+          "fee_recipient": "0x50155530FCE8a85ec7055A5F8b2bE214B3DaeFd3",
+          "builder": {
+            "enabled": true,
+            "gas_limit": "12345654321"
+          }
+        }
+      },
+      "default_config": {
+        "fee_recipient": "0x6e35733c5af9B61374A128e6F85f553aF09ff89A"
+        "builder": {
+          "enabled": false,
+        }
+      }
+    }
+    ```
+
+    In this example, validator `0xa057816...` is registered with the builder,
+    but any validator using the default configuration is not.
+
+!!! note
+
+    If you use a proposer configuration, you must enable blinded block production using [`--validators-proposer-blinded-blocks-enabled`](../Reference/CLI/CLI-Syntax.md#validators-proposer-blinded-blocks-enabled)
+
+### Example builder configurations
+
+!!! example "Validator client and beacon node in a single process"
+
+    ```bash
+    teku --validators-proposer-default-fee-recipient="0x6e35733c5af9B61374A128e6F85f553aF09ff89A" --ee-endpoint="http://127.0.0.1:8551" --ee-jwt-secret-file="/etc/jwt-secret.hex" --validators-builder-registration-default-enabled=true --builder-endpoint="http://127.0.0.1:18550"
+    ```
+
+!!! example "Validator client and beacon node in separate processes"
+
+    === "Validator client parameters"
+
+        ```bash
+        Teku validator-client --validators-proposer-blinded-blocks-enabled=true --validators-proposer-config="/etc/teku/proposerConfig.json"
+        ```
+
+    === "Proposer configuration"
+
+        ```json
+        {
+          "proposer_config": {
+            "0xa057816155ad77931185101128655c0191bd0214c201ca48ed887f6c4c6adf334070efcd75140eada5ac83a92506dd7a": {
+              "fee_recipient": "0x50155530FCE8a85ec7055A5F8b2bE214B3DaeFd3",
+              "builder": {
+                "enabled": false
+              }
+            }
+          },
+          "default_config": {
+            "fee_recipient": "0x6e35733c5af9B61374A128e6F85f553aF09ff89A",
+            "builder": {
+              "enabled": true
+            }
+          }
+        }
+        ```
+
+        In this example, MEV-Boost is enabled by default, and validator `0xa057...` is specifically disabled.
```

Comment:
I think this could be slightly confusing since it's different from the cli option configuration, I think we can just leave the enabled by default and remove entirely the `builder` field from the validator specific. We wouldnt want to disable it. We explain the difference a bit above anyways when describing the proposer config.